lib/rubycritic/commands/default.rb

```diff
@@ -12,7 +13,12 @@ def initialize(paths)
       end
 
       def execute
-        report(critique)
+        case Config.format
+        when :json
+          report(critique)
+        else
+          open(report(critique))
+        end
```

Comment:
I just merged the PR mentioned above, please rebase against master and then lets add handling :console_report here.